spec/db-seed/index.js

```diff
@@ -78,8 +78,8 @@ const generateRandomListing = () => {
   ];
   // Add the guitar's condition
   randomListing.condition = Math.floor(Math.random() * 9) + 1;
-  // Generate file names for 2 to 7 small photos for the listing
-  const randomNumberOfhotos = Math.floor((Math.random() * 6) + 2);
+  // Generate file names for 3 to 7 small photos for the listing
+  const randomNumberOfhotos = Math.floor((Math.random() * 5) + 3);
```

Comment:
Hotos or photos? If this is intentional it's fine - if not, you might want to rename it so you won't try calling the wrong variable down the line.

Reply:
Great catch with this one. I updated it! Thanks.